tests/unit/test_charm.py

```diff
@@ -189,3 +189,108 @@ def test_invalid_application_config(
     status = harness.model.unit.status
     assert isinstance(status, BlockedStatus)
     assert status.message == message
+
+
+@pytest.mark.parametrize("jvm_config", ["-Xmx1G", "-Xms200k", "-Xmx10m -Xms4096"])
+def test_jvm_config(harness: Harness, patch: SpringBootPatch, jvm_config):
+    """
+    arrange: provide a simulated Spring Boot application image.
+    act: update the jvm-config with a valid value.
+    assert: Spring Boot charm should start the charm with correct environment variables in the
+        pebble layer.
+    """
+    patch.start(
+        {"spring-boot-app": OCIImageMock.builder().add_file("/app/test.jar", b"").build()},
+        container_mock_callback={
+            "spring-boot-app": lambda container: container.process_mock.register_command_handler(
+                lambda command: command[0].endswith("java"),
+                lambda command, environment: (0, "", ""),
```

Comment:
would `lambda _, _` work in this case since the arguments are unused?

Reply:
Similar comment for other lambdas

Reply:
I just add argument names so reader can have a better understand what this lambda function does.